### DISCUSSION TITLE
How to remove gray from a black and white image

### DISCUSSION THREAD

**Discussion by Qlwentt** [ImageMagick/ImageMagick#7012] [Help] [answered]
### ImageMagick version

7.1.1-24

### Operating system, version and so on

Mac OS

### Description

Trying to take the gray out of this image: 
![temp_image](https://github.com/ImageMagick/ImageMagick/assets/16737433/c08c6419-2e39-4139-ac42-16738ba2fe0c)
like this (I used https://imgonline.tools/remove-color)
![temp_image (1)](https://github.com/ImageMagick/ImageMagick/assets/16737433/66d4a0fe-54e5-4e76-859f-31a13a09218f)

I tried this command I found here: https://www.imagemagick.org/discourse-server/viewtopic.php?t=36051

```
convert temp-image.png \( +clone -colorspace HCL -channel G -separate +channel -threshold 15% \) -alpha off -compose CopyOpacity -composite out.png
```

but that results in a blank image. I think because the image is in black and white and not color. Does anyone know how to do this?

**Comment by dlemstra** [comment]
The image is not gray it contains a lot of different colors? You want to change the image to black and white?

**Reply by fmw42** [reply, marked as answer]
There is no gray in that image.  What that code is doing is removing low saturation. The amount removed is determined by the threshold.  Try lowering the threshold value.

**Reply by Qlwentt** [reply]
thanks. lowering the threshold value worked!
